lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -473,7 +520,7 @@ def add_resources_bundle_targets(file_accessors)
 
                   # Set the `SWIFT_VERSION` build setting for resource bundles that could have resources that get
                   # compiled such as an `xcdatamodeld` file which has 'Swift' as its code generation language.
-                  if contains_compile_phase_refs && target.uses_swift?
+                  if contains_compile_phase_refs && file_accessors.any? { |fa| target.uses_swift_for_spec?(fa.spec) }
```

Comment:
Why would this be any target and not the spec of the `file_accessor` we are looping in? Isnt that where the resource bundle is under?

Alternatively I was under the impression from our offline chat that we would always set this without caring whether Swift is needed or not.

Reply:
> Why would this be any target and not the spec of the `file_accessor` we are looping in? Isnt that where the resource bundle is under?

I changed it to do just that

> Alternatively I was under the impression from our offline chat that we would always set this without caring whether Swift is needed or not.

Decided not to change that here, on the off chance it would accidentally break something